### PR TITLE
EasyToFly article

### PR DESCRIPTION
**Pull request by bart02**

### CODE REVIEW

**Comment by okalachev** [comment]
Would you like to provide the English version of the article also?

File: docs/ru/easytofly.md [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+
+Далее было написано программное обеспечение для работы с оборудованием. Было необходимо передать данные с микроконтроллера, установленного на Arduino Nano на микрокомпьютер, представить показания с датчиков в том же виде, в котором они представлены при получении данных с лидара. И после этого произвести те же действия, что и с лидаром.
+
+### Программные подсистемы «Мастер первоначальной настройки БПЛА» и «Монитор состояния БПЛА»
+
+Нами была запланирована разработка еще одной программной подсистемы, которая позволит настроить БПЛА, используя единый пользовательский веб-интерфейс, который «проведет» пользователя через весь процесс настройки. Также в данный веб-интерфейс должен быть встроен монитор текущего состояния БПЛА и редактор настроек.
+
+В первую очередь нами был сделан каркас будущего интерфейса.
+
+Также на данный момент реализован интерфейс калибровки датчиков БПЛА со встроенной анимацией для удобной работы с ним. Работу интерфейса калибровки можно увидеть на видео: [https://youtu.be/_XwKn4O9e48](https://youtu.be/_XwKn4O9e48).
```
Comment (okalachev) [inline docs/ru/easytofly.md:75]: Wouldn't embedding the video into the article with iframe better?

**Review by okalachev — CHANGES_REQUESTED** [review]

**Comment by okalachev** [comment]
Also, seems like the article lacks some images.

**Comment by okalachev** [comment]
Please, review my edits.

File: docs/ru/easytofly.md (hunk 1 above)
Comment (bart02) [reply docs/ru/easytofly.md]: Corrected https://github.com/CopterExpress/clover/pull/309/commits/9af85ae87650246da47f9878dc2f13faf9c8a57c

File: docs/ru/easytofly.md [hunk 2]
```diff
@@ -0,0 +1,142 @@
+## Информация о команде
+Название команды: EasyToFly
+
+В команде 5 человек: 
+- Игорь Сидорин [@maerans12](https://github.com/maerans12) (TeamLead)
```
Comment (okalachev) [inline docs/ru/easytofly.md:5]: Do I understand correctly `@maerans12` are Telegram usernames? I can make them links.

File: docs/ru/easytofly.md [hunk 3]
```diff
@@ -0,0 +1,142 @@
+## Информация о команде
```
Comment (okalachev) [inline docs/ru/easytofly.md:1]: Main title of the article missing (and this would be the name of your project).

File: docs/ru/easytofly.md [hunk 4]
```diff
@@ -0,0 +1,142 @@
...
+Название команды: EasyToFly
```
Comment (okalachev) [inline docs/ru/easytofly.md:2]: Please, provide e-mail, Telegram, of something for contact.

File: docs/ru/easytofly.md [hunk 5]
```diff
@@ -0,0 +1,142 @@
...
+2. Проведение анализа существующих решений
+3. Разработка технического задания (далее – ТЗ) на АПК
+4. Изучение литературы, необходимой для реализации проекта
+5. Формирование команды исполнителей на основании технического задания
+6. Разработка программной подсистемы АПК «Предотвращение столкновений»
+7. Разработка программной подсистемы АПК «Мастер первоначальной настройки БПЛА»
+8. Разработка программной подсистемы АПК «Монитор состояния БПЛА»
+9. Разработка аппаратной составляющей АПК
+
+![Лого](https://user-images.githubusercontent.com/19249148/89198142-edf20400-d5d6-11ea-9a83-c4ec9c989313.png)
```
Comment (okalachev) [inline docs/ru/easytofly.md:33]: Please, commit images to the repo. External images are not allowed.

File: docs/ru/easytofly.md [hunk 6]
```diff
@@ -0,0 +1,142 @@
...
+
+## Использование нашего продукта
+
+Репозиторий проекта доступен по ссылке https://github.com/easy-to-fly/easy-to-fly.
+
+### Сборка крепления для датчиков
+
+Вам понадобится:
+- **4x** стойки M3x6
+- **1x** крепаление для батареи (ССЫЛКА)
```
Comment (okalachev) [inline docs/ru/easytofly.md:98]: The link is missing.

**Review by okalachev — CHANGES_REQUESTED** [review]

**Comment by okalachev** [comment]
I made some technical edits to your article. Please provide me write access to your branch so I could push them. It's made with "Allow edits from maintainers" checkbox: https://docs.github.com/en/github/collaborating-with-issues-and-pull-requests/allowing-changes-to-a-pull-request-branch-created-from-a-fork.

Also, please resolve the issues I wrote above.

**Comment by bart02** [comment]
> I made some technical edits to your article. Please provide me write access to your branch so I could push them. It's made with "Allow edits from maintainers" checkbox: https://docs.github.com/en/github/collaborating-with-issues-and-pull-requests/allowing-changes-to-a-pull-request-branch-created-from-a-fork.
> 
> Also, please resolve the issues I wrote above.

There is not checkbox :( Maybe because it is organisation repo.
Btw try to commit now, add you as collaborator.

**Comment by okalachev** [comment]
I'm waiting for you to resolve the issues and merging.

File: docs/ru/easytofly.md (hunk 4 above)
Comment (bart02) [reply docs/ru/easytofly.md]: https://github.com/CopterExpress/clover/pull/309/commits/57373f3b3503b7c29e26c54b4fa3ecc237e76fe2

File: docs/ru/easytofly.md (hunk 5 above)
Comment (bart02) [reply docs/ru/easytofly.md]: https://github.com/CopterExpress/clover/pull/309/commits/f505bc5ca311624a68d606ece5e43ffb45b9ceee

File: docs/ru/easytofly.md (hunk 6 above)
Comment (bart02) [reply docs/ru/easytofly.md]: https://github.com/CopterExpress/clover/pull/309/commits/bce1b0960f7b7430141087de4afcf4068fac00d9

**Comment by bart02** [comment]
@okalachev Ready to merge

**Pull request merged**